src/rules/property-no-unknown/README.md

```diff
@@ -12,35 +12,37 @@ This rule considers properties defined in the [CSS Specifications and browser sp
 
 This rule ignores variables (`$sass`, `@less`, `--custom-property`).
 
+This rule ignores vendor prefixed properties (`-moz-align-self`, `-webkit-align-self`) .
```

Comment:
Let's add `.e.g.`, so (e.g., `moz-align-self` ...), so it's clear that this is not an exhaustive list.